snuba/migrations/snuba_migrations/functions/0001_functions.py

```diff
@@ -0,0 +1,189 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import (
+    UUID,
+    AggregateFunction,
+    Array,
+    Column,
+    DateTime,
+    Float,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+common_columns: List[Column[Modifiers]] = [
+    Column("org_id", UInt(64)),
+    Column("project_id", UInt(64)),
```

Comment:
WHy do you need both organization and project? Are you expecting to query across projects in an organization? It seems it would be weird as all queries are filtered by transaciton_name.

In practice, unless you are filtering thousands of specific project ids, filtering by a group of projects is more efficient than doing it by organization as you do not need to load an additional UInt64 column (organization).
It also keeps your primary key more efficient as there would be one fewer field.